xoinvader/level.py

```diff
@@ -1,17 +1,17 @@
-"""Module for creating and maintaining waves of incoming enemies."""
+"""Module for creating and maintaining xoinvader levels."""
 
-class EnemyWave(object):
-    """Container for enemy wave behavior.
+class Level(object):
```

Comment:
Any code except imports must left two empty lines from start of file / module docstring. Add one empty line)